### DISCUSSION TITLE
Публичные формы

### DISCUSSION THREAD

**Discussion by savosik** [moonshine-software/moonshine#1145] [Ideas]
Часто маркетологам нужно провести опрос, либо запросить реквизиты у клиента.   Можно добавить метод ресурсу ->withPublicForm()  который позволял бы получать ссылку на форму, которой можно было бы делиться, при этом форма по ссылке открывалась бы без авторизации, а так же без лэйаута.  Желательно, чтобы форма поддерживала вставку в iframe.

**Comment by lee-to** [comment]
Муншайн дает все возможности разработчику чтобы реализовать подобное
